checkov/terraform/graph_builder/local_graph.py

```diff
@@ -33,6 +34,7 @@
 
 MODULE_RESERVED_ATTRIBUTES = ("source", "version")
 CROSS_VARIABLE_EDGE_PREFIX = '[cross-variable] '
+FOREACH_STRING = 'for_each'
```

Comment:
Refactor `for_each` into consts file and reuse here and `checkov/terraform/graph_builder/foreach_handler.py`